.travis.yml

```diff
@@ -14,8 +14,14 @@ git:
 #  directories:
 #    - ${HOME}/.cache/pip
 
-# safelist
-# branches with associated PRs are still built by https://docs.travis-ci.com/user/pull-requests
+# run tests when changes (pushes) happen to these branches;
```

Comment:
do we still need the Travis-related files?

Reply:
I think we should keep Travis alive in tandem for at least a month. I'm pretty sure we can make it still run but be non-mandatory. See https://github.com/neuropoly/spinalcordtoolbox/issues/3071#issuecomment-751832655.

Reply:
> > Do we have a lot of users on Windows? Booting up WSL is still slower than anything else.
> 
> Hum, I really don't know. My guesstimate would be 50-200 Windows users.
> 
> > Maybe we could leave the WSL build to cron?

2abfcc6031a3b08d5419251d5d93c3a4b517674b